hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2141,6 +2141,12 @@
             - Write and read-check OTBN and IMEM for consistency.
             - Fetch a new key from the OTP_CTRL and ensure that previous contents in the IMEM and
               DMEM cannot be read anymore.
+
+            Note that fetching OTBN keys from OTP_CTRL requires OTP_CTRL to communciate with EDN.
+            And entropy_src, edn and csrng are initialized by the startup code in the test ROM
+            already.
+            There are assertion checks at the output of all EDN channels to verify validity of the
+            returned data.
```

Comment:
super nit: Could make this a bit more concise and in line with our bulleted list style:

First bullet in the list:  
```
            - Initialize the entropy_src subsystem to enable OTP_CTRL fetch random data (already
              done by the test_rom startup code).  
```

Add this as last bullet:
```
            - Check the validity of EDN's output to OTP_CTRL via assertions. 
```